charts/spire/README.md.gotmpl

```diff
@@ -49,6 +49,21 @@ spec:
         - --service-account-signing-key-file=/run/config/pki/sa.key
 ```
 
+## Deployment - Non Production
```

Comment:
Same commentary as before (but in the template)

I don't think we should lead with our least desirable deployment approach, we want to bury the items we don't want to be the primary approach after the first option.